bots/notify/src/main/java/org/openjdk/skara/bots/notify/issue/IssueNotifier.java

```diff
@@ -192,9 +192,10 @@ public void onIntegratedPullRequest(PullRequest pr, Path scratchPath, Hash hash)
 
             // If prOnly is false, this is instead done when processing commits
             if (prOnly && resolve) {
-                if (issue.state() == Issue.State.OPEN) {
+                if (issue.state() != Issue.State.OPEN) {
+                    log.info("Resolving issue which status is not open, the status is " + issue.state());
+                } else {
                     log.info("Resolving issue " + issue.id());
```

Comment:
No need for conditional here, we can use the same log message in regardless of current state. Also, updating assignees should also be done (as long as it's currently empty).
```suggestion
                log.info("Resolving issue " + issue.id() + " from state " + issue.state());
```